pom.xml

```diff
@@ -110,35 +110,40 @@
                 <!-- the Sling instance is provisioned from the model in src/main/provisioning/model.txt -->
                 <groupId>org.apache.sling</groupId>
                 <artifactId>slingstart-maven-plugin</artifactId>
-                <version>1.7.0</version>
+                <version>1.7.16</version>
                 <extensions>true</extensions>
                 <executions>
                     <execution>
                         <id>prepare-launchpad-package</id>
+                        <phase>pre-integration-test</phase>
                         <goals>
                             <goal>prepare-package</goal>
                         </goals>
                     </execution>
                     <execution>
                         <id>build-launchpad-package</id>
+                        <phase>pre-integration-test</phase>
                         <goals>
                             <goal>package</goal>
                         </goals>
```

Comment:
You should configure attachArtifact to false to prevent the IT launchpad from being deployed with the release.

Reply:
Right, that makes sense.